documentation/docs/documentation/06-generators/openapi/01-introduction.mdx

```diff
@@ -0,0 +1,61 @@
+---
+sidebar_position: 1
+title: Openapi generator introduction
+sidebar_label: Introduction
+---
+
+Note: Openapi Hyper Fetch code generator is still in beta. Please let us know in case of any problems.
+
+Note: Right now this code generator accepts only V3 version of the openapi schema.
+
+## Getting Started
+
+In order to start using the request generator, you need only one thing: OpenApi V3 JSON schema file or link to it. Then,
+you can simply call the package via `npx`:
+
+```tsx
+npx @hyper-fetch/codegen-openapi --schema https://petstore3.swagger.io/api/v3/openapi.json
+```
+
+This command will generate the `openapi.client.ts` that contains generated Hyper Fetch client, requests, and types:
+
+```tsx
+...
+export const client = new Client({ url: "/api/v3" });
```

Comment:
Does it work with this url? I mean, I would assume it will be full url not part of the path?

"/api/v3" => "https://-------.com/api/v3"

Reply:
It won't work with this url but it is not up to us what is the url in the openapi schema. I suppose we could add the option --baseUrl for the cli. What do you think?

Reply:
Good idea with url 👍